apps/reference/nav/_referenceSidebars.js

```diff
@@ -192,6 +192,7 @@ const sidebars = {
         'guides/platform/logs',
         'guides/platform/metrics',
         'going-into-prod',
+        'guides/platform/disk-usage',
         'guides/platform/performance',
         'guides/platform/permissions',
```

Comment:
We should default to alphabetical ordering (by sidebar label) unless there's a good reason otherwise.
```suggestion
        'guides/platform/disk-usage',
        'guides/platform/logs',
        'guides/platform/metrics',
        'guides/platform/performance',
        'guides/platform/permissions',
        'going-into-prod',
```